### PR TITLE
Add GA event for text selection

### PR DESCRIPTION
**Pull request by cdrini**

### CODE REVIEW

File: src/plugins/plugin.text_selection.js [hunk 1]
```diff
@@ -273,7 +274,20 @@ export class BookreaderWithTextSelection extends BookReader {
       // contains a reference to it.
       this.options.plugins.textSelection = options;
       this.textSelectionPlugin.init();
+
+      // Track how often selection is used
+      const sso = new SelectionStartedObserver('.textSelectionSVG', () => {
+        // Don't assume the order of the plugins ; the analytics plugin could
+        // have been added later. But at this point we should no for certain.
```
Comment (cdrini) [inline src/plugins/plugin.text_selection.js:281]:
```suggestion
        // have been added later. But at this point we should know for certain.
```

File: src/plugins/plugin.text_selection.js [hunk 2]
```diff
@@ -273,7 +274,20 @@ export class BookreaderWithTextSelection extends BookReader {
...
+        if (!this.archiveAnalyticsSendEvent) {
+          sso.detach();
+        } else {
+          this.archiveAnalyticsSendEvent?.('BookReader', 'SelectStart');
```
Comment (cdrini) [inline src/plugins/plugin.text_selection.js:285]:
```suggestion
          this.archiveAnalyticsSendEvent('BookReader', 'SelectStart');
```

File: src/plugins/plugin.text_selection.js [hunk 3]
```diff
@@ -273,7 +274,20 @@ export class BookreaderWithTextSelection extends BookReader {
...
+        }
+      });
+      sso.attach();
```
Comment (iisa) [inline src/plugins/plugin.text_selection.js:288]: do you need to pin this somewhere to detach later?

**Review by iisa — APPROVED** [review]
looks good just a question

File: src/plugins/plugin.text_selection.js (hunk 3 above)
Comment (cdrini) [reply src/plugins/plugin.text_selection.js:288]: I have! I detach it weirdly just above :P
